public/js/main.js

```diff
@@ -39,7 +39,8 @@
     }
 
     kitten_imagery.setAttribute('class', 'kitten')
-
+    kitten_imagery.setAttribute('href', window.kittens[index].url)
```

Comment:
Given the above discussion, all reference to the link will be removed. (this will nullify a few of your CR comments @c4rl)